### PR TITLE
Feature/highest lowest scoring visitor

### PR DESCRIPTION
**Pull request by austincarrjones**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [ ] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change

1. Added 2 methods: highest/lowest_scoring_visitor. Passing class tests. Haven't checked StatTracker. 

2. Noticing some merge issues on game_team_spec.rb, I'm guessing my file wasn't up to date with the latest PR. Can we resolve in the merge? Side by side comparison of the file seems clear enough to accept most of the old file, and then only update with my new tests on this branch. Or if it's easier for me to remove this PR, let me know. I'm just not sure what steps I would take on this branch to update locally before submitting another PR. Happy to hop on a call to review together.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before submitting PR
- [X] I assigned at least 2 group members to review my PR

### CODE REVIEW

File: spec/game_team_spec.rb [hunk 1]
```diff
@@ -133,126 +133,6 @@
       end
     end
 
-    describe "::team_id_and_shots - helper" do
```
Comment (zach-bergman) [inline spec/game_team_spec.rb:136]: Did you accidentally delete all of these tests? Or am I reading this wrong?

**Review by zach-bergman — COMMENTED** [review]
Just wondering if you deleted some tests from game_team_spec?

File: spec/game_team_spec.rb (hunk 1 above)
Comment (austincarrjones) [reply spec/game_team_spec.rb:136]: I have all of those tests on my main branch. Maybe I created the branch before pulling down main. I'm checking if it's possible to resolve in GitHub.

**Comment by austincarrjones** [comment]
There's no delete option, just close. I'll update my local with that game_team_spec, then create a new PR.

**Pull request closed without merging**